config.py

```diff
@@ -64,6 +64,7 @@ class Config(object):
     FEATURE_FLAGS_EDIT_SERVICE_PAGE = False
     FEATURE_FLAGS_GCLOUD7_OPEN = False
     FEATURE_FLAGS_G7_CLARIFICATIONS_CLOSED = False
+    FEATURE_FLAGS_SHOW_DOS_DECLARATION = False
```

Comment:
Could we have it "true" in Development?  Seems like we want to see all the stuff that's still being worked on?